files/zh-cn/web/css/css_grid_layout/grid_layout_and_progressive_enhancement/index.md

```diff
@@ -9,46 +9,21 @@ slug: Web/CSS/CSS_grid_layout/Grid_layout_and_progressive_enhancement
 
 ## 浏览器支持
 
-除了在 Internet Explorer 中，CSS Grid Layout 在 Safari，Chrome，Opera，Firefox 和 Edge 中都没有前缀。对这些指南中详述的所有属性和值的支持可跨浏览器进行互操作。这意味着如果您在 Firefox 中编写一些网格布局代码，它应该在 Chrome 中以相同的方式工作。这不再是实验规范，您可以安全地在生产环境中使用它。.
-
-## IE 和 Edge
-
-应当谨记 CSS 网格布局在 IE 上实现的最早版本是 IE10。这个早期规范并没有包含最新规范的所有属性和可用值。即使是相同的属性和相同的值，在 IE10 中的实现和现行的规范也有很大差异。早期规范也是 Edge15 之前实现的网格布局的版本。
-
-IE/Edge(≤15)版本的规范带有`-ms`前缀，IE/Edge(≤15) 上实现的属性如下：
-
-- {{cssxref("grid-template-columns")}} 变成 `-ms-grid-columns`
-- {{cssxref("grid-template-rows")}} 变成 `-ms-grid-rows`
-- {{cssxref("grid-row-start")}} 变成 `-ms-grid-row`
-- {{cssxref("grid-column-start")}} 变成 `-ms-grid-column`
-- {{cssxref("align-self")}} 变成 `-ms-grid-row-align`
-- {{cssxref("justify-self")}} 变成 `-ms-grid-column-align`
-
-IE 版本另有一些在新版规范中非必须的属性`-ms-grid-column-span`和`-ms-grid-row-span`。这个版本不包含自动定位能力和网格模板区域。从 IE10 到 Edge15 的版本，使用`-ms`属性可以实现一些简单的网格布局。由于这些属性包含厂商前缀，它们在任何支持最新规范使用无前缀属性的浏览器上将无法生效。
-
-### Autoprefixer 对网格布局的支持
-
-流行的*[Autoprefixer](https://github.com/postcss/autoprefixer)*更新之后已经能够支持`-ms-`网格布局版本。网格前缀默认是禁用的，你可以通过`grid: true`选项启用它。
-
-```
-autoprefixer({ grid: 'autoplace' })
-```
-
-Grid 网格前缀被默认禁用是因为存在一些属性不能添加前缀。
+CSS Grid Layout 在 Safari，Chrome，Opera，Firefox 和 Edge 中都没有前缀。对这些指南中详述的所有属性和值的支持可跨浏览器进行互操作。这意味着如果您在 Firefox 中编写一些网格布局代码，它应该在 Chrome 中以相同的方式工作。这不再是实验规范，您可以安全地在生产环境中使用它。
```

Comment:
Please use "你" instead of "您", and update the translation of the whole paragraph. (Check the [translation-guide](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E4%BB%A3%E5%90%8D%E8%AF%8D).)

```suggestion
CSS Grid Layout 在所有现代浏览器中都没有前缀。对这些指南中详述的所有属性和值的支持可跨浏览器进行互操作。这意味着如果你在 Firefox 中编写一些网格布局代码，它应该在 Chrome 中以相同的方式工作。这不再是实验规范，你可以安全地在生产环境中使用它。
```

Reply:
Done. Please review.